src/database/firebase.tsx

```diff
@@ -95,6 +95,34 @@ export const addTree = async (tree: Tree) => {
   }
 };
 
+/**
+ * same functionality as addTree but for comments:
+ * assigns uuid to comment and adds to commentCollection.
+ */
+export const saveComment = async (comment: Comment) => {
+  try {
```

Comment:
we don't need a comment collection if we are nesting our comments in our tree document itself @kennethlien  thoughts

Reply:
ya since they're going to be in each tree doc I don't think we need this complete collection of all comments in the firestore